core/domain/suggestion_registry.py

```diff
@@ -443,20 +530,25 @@ def populate_old_value_of_change(self):
             old_content = (
                 exploration.states[self.change.state_name].content.to_dict())
 
-        self.change.old_value = old_content
+        # Here, change is of type ExplorationChange and all attributes
+        # on ExplorationChange are created dynamically except cmd, so due
+        # this MyPy is unable to recognize `old_value` as an attribute
+        # of change and throwing `"ExplorationChange" has no attribute
```

Comment:
```suggestion
        # on ExplorationChange are created dynamically except cmd, so because
        # of this MyPy is unable to recognize `old_value` as an attribute
        # of change and throws `"ExplorationChange" has no attribute
```

Reply:
Done!